### DISCUSSION TITLE
Tabular Editor 2 freezing after any changes

### DISCUSSION THREAD

**Discussion by VasilinaBalashevich** [TabularEditor/TabularEditor3#1154] [🙋♀️TE3 Q&A]
When I do any changes in a tabular cube, like adding a new table or even renaming the existing ones, the TE starts freezing for ages (  about 1 hour). I've reinstalled it several times, the service desk team updated the system, so it must work properly, as my mates have the same laptops and the same version of TE, and they don't face this issue. Please,  give me a hand as it hinders efficient work.

**Comment by otykier** [comment]
Just to clarify, does this freeze happen when you attempt to save the change, or does it happen immediately when you make the change? Which version of TE2 are you using?

**Reply by VasilinaBalashevich** [reply]
It happens immediately when I make the change, e g I have just created a new relationship, once I clicked 'new relationship' it started freezing. the version is TE 2.20.2

**Reply by otykier** [reply]
I'm sorry, but it's not something I've seen or heard about before, so it's not easy to figure out what's causing this. Are your colleagues using the same version? Do you have access to another machine where you could try to install Tabular Editor to see if the same issue occurs there?

**Reply by VasilinaBalashevich** [reply]
it is really frustrating... yes, the colleagues use the same version. Unfortunately, I don't have another machine, probably it makes sense to install it on VM?

**Comment by otykier** [comment]
Since this issue is regarding Tabular Editor 2, I have created an issue ticket to track it here: https://github.com/TabularEditor/TabularEditor/issues/1148
